# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnome-python-extras/gdl-api.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

65dc5cb96414698b Mon Sep 17 00:00:00 2001
From: Gian Mario Tagliaretti <gianmt@gnome.org>
Date: Thu, 28 May 2009 20:40:05 +0200
Subject: [PATCH] Remove API and objects no more shipped with gld

---
 gdl/gdl.defs |   81 ----------------------------------------------------------
 1 files changed, 0 insertions(+), 81 deletions(-)

diff --git a/gdl/gdl.defs b/gdl/gdl.defs
index 5babad1..085de39 100644
--- a/gdl/gdl.defs
+++ b/gdl/gdl.defs
@@ -46,13 +46,6 @@
   (gtype-id "GDL_TYPE_DOCK_ITEM")
 )
 
-(define-object DockNotebook
-  (in-module "Gdl")
-  (parent "GdlDockItem")
-  (c-name "GdlDockNotebook")
-  (gtype-id "GDL_TYPE_DOCK_NOTEBOOK")
-)
-
 (define-object Dock
   (in-module "Gdl")
   (parent "GdlDockObject")
@@ -60,13 +53,6 @@
   (gtype-id "GDL_TYPE_DOCK")
 )
 
-(define-object DockPaned
-  (in-module "Gdl")
-  (parent "GdlDockItem")
-  (c-name "GdlDockPaned")
-  (gtype-id "GDL_TYPE_DOCK_PANED")
-)
-
 (define-object DockPlaceholder
   (in-module "Gdl")
   (parent "GdlDockObject")
@@ -74,12 +60,6 @@
   (gtype-id "GDL_TYPE_DOCK_PLACEHOLDER")
 )
 
-(define-object DockTablabel
-  (in-module "Gdl")
-  (parent "GtkBin")
-  (c-name "GdlDockTablabel")
-  (gtype-id "GDL_TYPE_DOCK_TABLABEL")
-)
 
 (ifdef HAVE_GDL_0_7
 
@@ -682,21 +662,6 @@
 
 
 
-;; From gdl-dock-notebook.h
-
-(define-function gdl_dock_notebook_new
-  (c-name "gdl_dock_notebook_new")
-  (is-constructor-of "GdlDockNotebook")
-  (return-type "GtkWidget*")
-)
-
-(define-function gdl_dock_notebook_get_type
-  (c-name "gdl_dock_notebook_get_type")
-  (return-type "GType")
-)
-
-
-
 ;; From gdl-dock-object.h
 
 (define-function gdl_dock_object_get_type
@@ -848,24 +813,6 @@
 
 
 
-;; From gdl-dock-paned.h
-
-(define-function gdl_dock_paned_get_type
-  (c-name "gdl_dock_paned_get_type")
-  (return-type "GType")
-)
-
-(define-function gdl_dock_paned_new
-  (c-name "gdl_dock_paned_new")
-  (is-constructor-of "GdlDockPaned")
-  (return-type "GtkWidget*")
-  (properties
-    '("orientation")
-  )
-)
-
-
-
 ;; From gdl-dock-placeholder.h
 
 (define-function gdl_dock_placeholder_get_type
@@ -896,34 +843,6 @@
 
 
 
-;; From gdl-dock-tablabel.h
-
-(define-function gdl_dock_tablabel_new
-  (c-name "gdl_dock_tablabel_new")
-  (is-constructor-of "GdlDockTablabel")
-  (return-type "GtkWidget*")
-  (properties
-    '("item")
-  )
-)
-
-(define-function gdl_dock_tablabel_get_type
-  (c-name "gdl_dock_tablabel_get_type")
-  (return-type "GType")
-)
-
-(define-method activate
-  (of-object "GdlDockTablabel")
-  (c-name "gdl_dock_tablabel_activate")
-  (return-type "none")
-)
-
-(define-method deactivate
-  (of-object "GdlDockTablabel")
-  (c-name "gdl_dock_tablabel_deactivate")
-  (return-type "none")
-)
-
 ;; From gdl-icons.h
 
 (ifdef HAVE_GDL_0_7
-- 
1.6.0.4

